testsuite/testsuite.c

```diff
@@ -241,7 +256,7 @@ static int test_tls(func_args* server_args)
 {
     func_args echo_args;
     char* myArgv[NUMARGS];
-    char arg[3][32];
+    char arg[3][128];
```

Comment:
Curious why the arg size increase?

Reply:
to accommodate `$TMPDIR`.  ran into the failure mode empirically :-)